packages/ui/src/ui/containers/OpenQueryButtons/OpenQueryButtons.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import cn from 'bem-cn-lite';
+import {Button} from '@gravity-ui/uikit';
+import {useDispatch, useSelector} from 'react-redux';
+
+import Icon from '../../components/Icon/Icon';
+import {YQLKitButton} from '../../containers/YQLKitButton/YQLKitButton';
+import {useQueryWidgetSidePanel} from '../../pages/query-tracker/QueryWidget/side-panel';
+import {QueryEngine} from '../../pages/query-tracker/module/engines';
+import {createQueryFromTablePath} from '../../pages/query-tracker/module/query/actions';
+import {createNewQueryUrl} from '../../pages/query-tracker/utils/navigation';
+import {getNavigationSqlService} from '../../store/selectors/settings/navigation';
+
+import './OpenQueryButtons.scss';
+
+const b = cn('yt-open-query-buttons');
+
+export type OpenQueryButtonProps = {
+    className?: string;
+    path: string;
+    cluster: string;
+};
+
+export function OpenQueryButtons({className, path, cluster}: OpenQueryButtonProps) {
+    const dispatch = useDispatch();
+    const {openWidget, widgetOpened, widgetContent} = useQueryWidgetSidePanel();
+
+    const {isQtKitEnabled} = useSelector(getNavigationSqlService);
+
+    const handleOpen = React.useCallback(() => {
+        dispatch(createQueryFromTablePath(QueryEngine.YQL, cluster, path));
+        openWidget();
+    }, [dispatch, path, cluster, openWidget]);
+
+    return !isQtKitEnabled ? null : (
```

Comment:
This isn't a bug, but I wouldn't put the c condition here `!`
Readability is slightly better without it